dynamiqs/time_array.py

```diff
@@ -167,6 +186,9 @@ class TimeArray(eqx.Module):
         mT _(TimeArray)_: Returns the time-array transposed over its last two
             dimensions.
         ndim _(int)_: Number of dimensions.
+        discontinuity_ts _(Array | None)_: Times at which there is a discontinuous jump
+            in the time-array values (the array is always sorted, but does not
+            necessarily contain unique values).
```

Comment:
Related to the main comment. I don't think the array is necesseraly sorted at this point.